sections/featured-blog.liquid

```diff
@@ -6,12 +6,29 @@
   {% endif %}
 
   {%- assign blog = blogs[section.settings.blog] -%}
-
+  
+  {% comment %}Assign articles to a variable for reuse{% endcomment %}
+  {% assign articles = blog.articles %}
+  
+  {% comment %}Iterate through articles to check for one containing the tag "first" (Added in the admin panel for blog posts)
+  			   and assign to a variable
+  {% endcomment %}
+  {% for article in articles %}
+  	{% if article.tags.first %}
+  		{% assign first = article %}
+  	{% endif %}	
+  {% endfor %}
+  {% comment %}Add the initial list of articles to the end of the declared first article, and then filter out the duplicated article{% endcomment %}
+  {% assign sortedArticles = first | concat: articles | uniq %}
```

Comment:
There should be a check to make sure a post with a "first" tag exists in the first place. If there isn't this code would probably fail.